sig-and-wg/sig-core/proposals/knative_function_integration.md

```diff
@@ -0,0 +1,361 @@
+# Integration of the knative based function controller
+
+## Introduction
+
+One of the main features of kyma is its ability to run serverless functions. Currently the functions provider in Kyma is the open source project [kubeless](https://github.com/kubeless/kubeless). With the focus shift towards knative, Kubeless will be replaced with a functions runtime controller (called `function-controller` from here on) that is the provider of a [Knative](https://github.com/knative) based functions (called `knative-functions` from here on).
+
+The migration to a new functions provider should provide a similar user experience as with the currently available kubeless runtime. This includes UI based CRUD operations for the functions as well as monitoring, alerting and  backup / restore.
+
+## Features
+
+### Current Features
+
+- As a developer, I want to provide my code as a function of a certain language and runtime to be deployed.
```

Comment:
We have got only node6 and node8. Should we mention that ? `certain language` is generic imo

Reply:
this is the generic description of the feature, regardless of what are the currently supported runtimes. We can add in the proposal that we only support 2 runtimes.